README.md

```diff
@@ -115,7 +115,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
 
-      IQKeyboardManager.shared.enable = true
+      IQKeyboardManager.shared().isEnabled = true
```

Comment:
It looks like you are suggesting the older API. The demo project is using the current one.